src/core/dbus-service.c

```diff
@@ -294,6 +294,29 @@ int bus_service_method_dump_file_descriptor_store(sd_bus_message *message, void
         return sd_bus_send(NULL, reply, NULL);
 }
 
+#if __SIZEOF_SIZE_T__ == 8
+static int property_get_size_as_uint32(
```

Comment:
I'd like to move this to bus-get-properties.c.

Reply:
dunno, i was unsure about that. it's somewhat generic, true, but it's also a nichey, very specific hack. So I thought leave it local for now until we maybe need this somewhere else too, in which case we can move it over?

Reply:
OK. That's not strong opinion.